doc: improve details around tagging CMakeLists.txt (#5192)
nrfconnect/sdk-nrf: doc/nrf/gs_modifying.rst

Maintaining CMakeLists.txt in SES
=================================

> :file:`CMakeLists.txt` files must be tagged properly before they can be added to a project in SES.

Reviewer: ```suggestion
You must tag the :file:`CMakeLists.txt` files properly before adding them to a project in SES.
```

> Projects in :file:`sdk-nrf` already have these tags, but projects from Zephyr and other repos do not.

Reviewer: ```suggestion
Projects in the :file:`sdk-nrf` repository already have these tags, but projects from Zephyr and other repositories do not.
```

> Follow the steps in :ref:`ses_tags_in_CMakeLists` to manually add tags when using :file:`CMakeLists.txt` files that are not located in the :file:`sdk-nrf` repo.

Reviewer: ```suggestion
Follow the steps in :ref:`ses_tags_in_CMakeLists` to manually add tags when using :file:`CMakeLists.txt` files that are not located in the :file:`sdk-nrf` repository.
```

* ``target_include_directories``
* ``target_compile_options``

> The :file:`CMakeLists.txt` files for the sample applications in :file:`sdk-nrf` already have the required tags.

Reviewer: ```suggestion
The :file:`CMakeLists.txt` files for the sample applications in the :file:`sdk-nrf` repository already have the required tags.
```
